src/main/java/ru/itis/security/util/JwtTokenUtil.java

```diff
@@ -0,0 +1,61 @@
+package ru.itis.security.util;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Component
+public class JwtTokenUtil implements Serializable {
+
+    private static final long JWT_TOKEN_VALIDITY = 86400;
+    private static final String SECRET = "ainaisthebest";
```

Comment:
It is not so secret, I can see it.

Reply:
in enterprise repos are private, so it's a secret